### DISCUSSION TITLE
Tower Defense - Risk Management

### DISCUSSION THREAD

**Discussion by Freaky2164** [Freaky2164/TowerDefense#36] [General]
Welcome back everybody,

we had a great winterbreak and now we are back in buiss. We are exited that we updatet the [README](https://github.com/argastle/TowerDefense/blob/main/README.md). Check it out:

Today we created a table for our [Risk Management](https://github.com/Freaky2164/TowerDefense/blob/main/Projektmanagement/Risk%20Management/Risk_Management.xlsx), you can see it here:

It lists six potential risks, including technical issues with the Unity engine, difficulty integrating artwork, and inadequate instructions for gameplay. Each risk is evaluated using a probability and impact score, with the resulting risk score determining its priority. Mitigation strategies and contingency plans are provided for each risk, along with an assigned responsible party and current status.

As the biggest risk, we see the missing resources to finish the projekt in all its glory, because there is much potential, that we can loose us in details and forget the bigger picture.

We also have some new ideas for our game, but at the moment it will be a secret, which will be revealed in the next weeks.
Please give us valueable feedback, so we can improve even more.

Greetings,
TD Team

<br>
<br>

[Sneak Peek](https://github.com/argastle/TowerDefense/blob/main/Projektmanagement/Mock-Ups/Path.png)

**Comment by schw1erig** [comment]
Hi TD Team,

Your risk management table provides a clear overview of the risks, their probability, impact scores, the mitigation strategies and contingency plans in place.
I'm looking forward to see what your teased ideas are.

Greetings
The Common Team

**Reply by WeberWr** [reply]
Hey Common Team,

Thanks for your comment. We are glad to hear, that the risk management is clear enough. 
Be aware of the updates in the next weeks to get more details about our game 

Kind regards,
TD

**Comment by KevStr06** [comment]
Hello TD team,

Your risk management looks good. In particular, I think your risk mitigation strategies and related indicators are well developed. But I hope you don't need your contingency plans.

Team cybuy

**Reply by WeberWr** [reply]
Hey cybuy,

thanks for the effort review our blog.
I think till now we are on a good way that we don't need our plans. But you should always be aware of the risks.

Regards
TD

**Comment by Fullstack-DEVil** [comment]
Hi Tower Defense,

nice to see that you are hard at work again. As I can see from your article and the attached table, you have thoroughly dealt with the topic of risk management and worked out the potential dangers. It is important that you have a plan that allows you to respond quickly and smoothly to any risk that may arise.

Best regards
Rule The Gym - Team
